ai/player_utils.py

```diff
@@ -14,11 +14,9 @@ def player_info(player_id, game_info):
     return game_info['known_info'][player_id]
 
 def is_safe(card, game_info):
+    [color, rank] = list(card)
```

Comment:
Hell yea. Looking at the card rather than the tableau is the right way to do this. I can be a dummy sometimes